oauth2client/contrib/appengine.py

```diff
@@ -56,6 +49,8 @@
 
 logger = logging.getLogger(__name__)
 
+Credentials = client.Credentials
```

Comment:
Left overs, from when there were indirect references.  There were some references in the code to `contrib.appengine.Credentials` which was just `client.Credentials`.

Removing.